shared.go

```diff
@@ -320,6 +324,26 @@ func (parsed rawHeader) getCritical() ([]string, error) {
 	return q, nil
 }
 
+// getS2C extracts parsed "p2c" from the raw JSON.
+func (parsed rawHeader) getP2C() (int, error) {
+	v := parsed[headerP2C]
+	if v == nil {
+		return 0, nil
```

Comment:
I think this should probably return an error here.

Reply:
I was following the behavior of similar methods on the rawHeader type, `getString()`, `getByteBuffer()`, `getJWK()`, ...